css/form.css

```diff
@@ -1,5 +1,6 @@
 .fl-form {
   position: relative;
+  padding: 15px;
```

Comment:
It's very unlikely that in order to add a new picture choice field, the entire form container needs a 15px padding around. This will introduce a breaking change to all the Form instances from our users.